src/configs/lang.ts

```diff
@@ -523,6 +523,11 @@ export const FILTER = translate('FILTER', 'Filter');
 
 export const USER = translate('USER', 'user');
 
+export const MDA_POINT_SCHOOL_REPORT_TITLE = translate(
+  'MDA_POINT_SCHOOL_REPORT_TITLE',
+  'MDA Point School Reporting'
+);
```

Comment:
We need to have two sets of labels.  One would be something like "MDA Point Location Report", and the other one should be "MDA Point School Report".  Then via a config we would use "MDA Point School Report" for MDA Eswatini but by default it should be "MDA Point Location Report".

We have already done this for the page that uploads MDA CSV, so you can take a look at that.